api-scripts/notes/mod-notes_GET_POST.jmx

```diff
@@ -0,0 +1,398 @@
+          <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
+          <stringProp name="HTTPSampler.connect_timeout"></stringProp>
+          <stringProp name="HTTPSampler.response_timeout"></stringProp>
+        </HTTPSamplerProxy>
+        <hashTree>
+          <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion statusCode 200" enabled="true">
+            <collectionProp name="Asserion.test_strings">
+              <stringProp name="49586">200</stringProp>
+            </collectionProp>
+            <stringProp name="Assertion.custom_message"></stringProp>
+            <stringProp name="Assertion.test_field">Assertion.response_code</stringProp>
+            <boolProp name="Assertion.assume_success">false</boolProp>
+            <intProp name="Assertion.test_type">16</intProp>
+          </ResponseAssertion>
+          <hashTree/>
+          <JSONPathAssertion guiclass="JSONPathAssertionGui" testclass="JSONPathAssertion" testname="JSON Assertion" enabled="true">
+            <stringProp name="JSON_PATH">$.title</stringProp>
+            <stringProp name="EXPECTED_VALUE">BU Campus Access Issues</stringProp>
+            <boolProp name="JSONVALIDATION">false</boolProp>
+            <boolProp name="EXPECT_NULL">false</boolProp>
+            <boolProp name="INVERT">false</boolProp>
+            <boolProp name="ISREGEX">true</boolProp>
+          </JSONPathAssertion>
+          <hashTree/>
+        </hashTree>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="mod-notes POST" enabled="true">
+          <boolProp name="HTTPSampler.postBodyRaw">true</boolProp>
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">{&#xd;
+  &quot;type&quot;: &quot;High Priority&quot;,&#xd;
+  &quot;title&quot; : &quot;BC Campus Access Issues&quot;,&#xd;
+  &quot;domain&quot;: &quot;eholdings&quot;,&#xd;
+  &quot;content&quot;: &quot;There have been access issues at the BU campus since the weekend&quot;,&#xd;
+  &quot;creator&quot;: {&#xd;
+    &quot;lastName&quot; : &quot;PTF&quot;,&#xd;
+    &quot;firstName&quot;: &quot;PTF&quot;,&#xd;
+    &quot;middleName&quot;: &quot;PTF&quot;&#xd;
+  },&#xd;
+  &quot;updater&quot;: {&#xd;
+    &quot;lastName&quot; : &quot;PTF&quot;,&#xd;
+    &quot;firstName&quot;: &quot;PTF&quot;,&#xd;
+    &quot;middleName&quot;: &quot;PTF&quot;&#xd;
+  },&#xd;
+  &quot;status&quot; : &quot;ASSIGNED&quot;,&#xd;
+  &quot;links&quot;: [&#xd;
+    {&#xd;
+      &quot;id&quot; : &quot;583-2356521&quot;,&#xd;
+      &quot;type&quot;: &quot;package&quot;&#xd;
+    },&#xd;
+    {&#xd;
+      &quot;id&quot; : &quot;583-2356521-758038&quot;,&#xd;
```

Comment:
POST request body is hardcoded. Is it required?

Reply:
The POST body is required to create a new record. It contains required fields and optional fields to give the record some size.

Reply:
Ok. In a real scenario, all notes will not be the same. Not sure if creating all these notes with unique content will show true performance as in a production environment. If it doesn't matter then it should be fine.